docs/servicenow.md

```diff
@@ -0,0 +1,143 @@
+# ticketutil.servicenow
+
+This document contains information on the methods available when working
+ with a ServiceNow object. A list of the ServiceNow fields that have been
+ tested when creating and editing tickets will be included. Because each
+ instance of ServiceNow can have custom fields and custom values, some
+ of the tested fields may not be applicable to certain instances of ServiceNow.
+ Additionally, your ServiceNow instance may contain ticket fields that we have
+ not tested. Keep in your mind different ServiceNow tables contain different
+ fields with different properties. Mandatory field of one table is optional
+ in another table (or it doesn't exist at all).
+
+Custom field names and values can be passed in as keyword arguments when
+ creating and editing tickets, and the ServiceNow REST API should be able
```

Comment:
well... I will rephrase it. The thing is there is no official list of fields on the Wiki, but there is "Introduction to Fields" and "Dot-Walking" which provides some information. Other way is to get_ticket_content which returns data of the ticket with specified ticket_id. I will mention this in the documentation.